ls1intum/Artemis
# Disable Exam Start button until 5 minutes before exam start

### Checklist

### Motivation and Context


Currrently students can click the exam start button any time once the exam is set visible. This is a potential security risk, so we want to only enable the exam start button 5 minutes before the actual exam start date. 
### Description

The exam start button for students is only enabled 5minutes before exam start. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create an exam 
4. Check if the button is only enabled 5 minutes before exam start. 
5. For test run exams this logic does not apply. 

### Test Coverage




### Screenshots
